src/month_view.es

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import clsx from 'clsx';
+import {
+  arrayOf,
+  getISODate,
+  getMonthDetails,
+  getISOMonth,
+  pad,
+  toDate,
+} from './utils';
+
+const DAYS_COUNT = 7;
+
+const renderLabelsRow = (locale, theme) => {
+  const { firstDay, weekdayShortLabels } = locale;
+  const labels = arrayOf(DAYS_COUNT).map((i) => {
+    const key = i + firstDay > 6 ? 0 : i + firstDay;
+    const className = clsx(theme.weekdayLabel, {
+      [theme.isWeekend]: key === 6 || key === 0,
+    });
+    return <th key={key} className={className}>{weekdayShortLabels[key]}</th>;
+  });
+  return <tr>{labels}</tr>;
+};
+
+const renderMonth = ({ theme, month, minDate, maxDate, selected, onCellClick }) => {
+  const { days, offset } = getMonthDetails(month);
+
+  const selectedISO = selected ? getISODate(selected) : null;
+  const monthISO = getISOMonth(month);
+  const minDateISO = minDate ? getISODate(minDate) : null;
+  const maxDateISO = maxDate ? getISODate(maxDate) : null;
+
+  const weeks = Math.ceil((days + offset) / DAYS_COUNT);
+  const today = getISODate(new Date());
+
+  const renderWeek = (week) => {
+    const shift = week * DAYS_COUNT;
+    const isFirstWeek = week === 0;
+
+    const items = arrayOf(DAYS_COUNT).map((day) => {
+      const date = shift + day + 1 - offset;
+
+      const key = `${monthISO}-${pad(date)}`;
+
+      const isBeforeMinDate = minDateISO ? key < minDateISO : false;
+      const isAfterMaxDate = maxDateISO ? key > maxDateISO : false;
```

Comment:
this is not a reliable way to compare dates. you could write a util that would parse all date parts as numbers and compare that (fast), or convert to date objects (slow)